commit 3b107e17f43f4696be2ecd53fc2587a9dfcddd10
Author: Jerry Jelinek <jerry.jelinek@joyent.com>
Date:   2019-05-17T20:21:25+00:00 (5 months ago)
    
    OS-7803 lp build broken by SMT changes
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Approved by: Patrick Mooney <patrick.mooney@joyent.com>

diff --git a/usr/src/cmd/lp/cmd/lpadmin/do_printer.c b/usr/src/cmd/lp/cmd/lpadmin/do_printer.c
index c26c59e39a..6b2472f175 100644
--- a/usr/src/cmd/lp/cmd/lpadmin/do_printer.c
+++ b/usr/src/cmd/lp/cmd/lpadmin/do_printer.c
@@ -40,9 +40,6 @@
 #include <bsm/devices.h>
 #include "lp.h"
 #include "class.h"
-#if defined PS_FAULTED
-#undef	PS_FAULTED
-#endif
 #include "printers.h"
 #include "msgs.h"
 
diff --git a/usr/src/cmd/lp/cmd/lpsched/exec.c b/usr/src/cmd/lp/cmd/lpsched/exec.c
index 4bd1da4a91..521a55eaa1 100644
--- a/usr/src/cmd/lp/cmd/lpsched/exec.c
+++ b/usr/src/cmd/lp/cmd/lpsched/exec.c
@@ -29,9 +29,6 @@
 
 #include <pwd.h>
 #include <zone.h>
-#if defined PS_FAULTED
-#undef  PS_FAULTED
-#endif /* PS_FAULTED */
 #include <dial.h>
 
 #include <stdlib.h>
diff --git a/usr/src/cmd/lp/include/printers.h b/usr/src/cmd/lp/include/printers.h
index baf8698645..312ca93cf1 100644
--- a/usr/src/cmd/lp/include/printers.h
+++ b/usr/src/cmd/lp/include/printers.h
@@ -25,13 +25,22 @@
 /*
  * Copyright 2006 Sun Microsystems, Inc.  All rights reserved.
  * Use is subject to license terms.
+ * Copyright 2019 Joyent, Inc.
  */
 
-#pragma ident	"%Z%%M%	%I%	%E% SMI"
-
 #if	!defined(_LP_PRINTERS_H)
 #define	_LP_PRINTERS_H
 
+/*
+ * The following conflicts are also defined in sys/processor.h.
+ */
+#if defined PS_FAULTED
+#undef	PS_FAULTED
+#endif
+#if defined PS_DISABLED
+#undef	PS_DISABLED
+#endif
+
 /*
  * Define the following to support administrator configurable
  * streams modules:
